src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -646,8 +655,12 @@ await using (var context = InMemoryApplicationDbContext(contextId))
                         TopicId = topic.Id
                     }
                 );
+
+                // NOTE: Not called because publication.Live is false
+                publicBlobCacheService.Verify(mock => mock.DeleteItem(It.IsAny<PublicationCacheKey>()),
```

Comment:
If you use strict behaviour on all the mocks you can get away without this here and in other places throughout the tests. If something unexpected is called it will fail the tests which is why we use the strict behaviour (wish this was the default behaviour!).

Reply:
I've switched them all to Strict, but I am actually inclined to think using MockBehavior.Default in these instances actually made it easier for someone to read what is going on. I've left a comment in the tests about why no Setup for publicBlobCacheService is necessary.